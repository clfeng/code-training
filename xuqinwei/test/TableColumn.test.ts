import { shallowMount } from "@vue/test-utils";
import { expect, test, vi } from "vitest";
import SimpleColumn from "../src/components/simpleTable/components/simpleColumn";


test("Table column", () => {
  const wrapper = shallowMount(SimpleColumn, {
    props: {
      prop: {}
    }
  }
  );

  expect(wrapper.html()).toMatchSnapshot();
});

