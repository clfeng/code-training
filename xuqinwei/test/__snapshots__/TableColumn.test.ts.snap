// Vitest Snapshot v1

exports[`Table column 1`] = `""`;

exports[`mount component 1`] = `""`;
